app/controllers/user_journey_controller.rb

```diff
@@ -88,17 +88,19 @@ def confirm
       component = klass_component(@upload.component_type).find_by_id(@upload.component_id)
 
       if @upload.certificate.encryption?
-        replace_event = ReplaceEncryptionCertificateEvent.create(
+        ReplaceEncryptionCertificateEvent.create(
```

Comment:
Is it ok to remove check metadata publish on the replace event.

Reply:
well spotted, I missed it.. Yeah, the replace event should be checked too

Reply:
thanks! I fell victim to the stubbing and the auto linting with this I am afraid. Have amended.